### PR TITLE
changed functionbeat config, and added new variable for logstash url

### PR DESCRIPTION
**Pull request by wanieldilson**

### CODE REVIEW

File: buildspec.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ env:
     TF_VAR_ost_username:                     "/codebuild/pttp-ci-infrastructure-core-pipeline/$ENV/ost_username"
     TF_VAR_ost_password:                     "/codebuild/pttp-ci-infrastructure-core-pipeline/$ENV/ost_password"
     TF_VAR_ost_url:                          "/codebuild/pttp-ci-infrastructure-core-pipeline/$ENV/ost_url"
+    TF_VAR_ost_logstash_url:                 "/codebuild/pttp-ci-infrastructure-core-pipeline/development/ost_logstash_url"
```
Comment (elcorbs) [inline buildspec.yml:28]: Should this be $ENV so you don't have to open a PR to push to pre-prod and prod?

File: modules/function_beats_config/main.tf [hunk 2]
```diff
@@ -166,16 +166,20 @@ locals {
...
-      password : var.destination_password
-      index : "functionbeat-%%{[agent.version]}"
-    }
-    "output.elasticsearch.ssl.certificate_authorities" : ["elk-ca.crt"]
-    "output.elasticsearch.ssl.certificate" : "moj.crt"
-    "output.elasticsearch.ssl.key" : "moj.key"
+    "output.logstash" : {
+      hosts: [var.ost_logstash_url]
+     }
+    # "output.elasticsearch" : {
```
Comment (elcorbs) [inline modules/function_beats_config/main.tf:172]: Do you need to leave the comments in? We should be able to find this stuff in git history if we need it again

File: buildspec.yml (hunk 1 above)
Comment (wanieldilson) [reply buildspec.yml:28]: Yes, yes it absolutely should

File: modules/function_beats_config/main.tf (hunk 2 above)
Comment (wanieldilson) [reply modules/function_beats_config/main.tf]: Done 👍

**Review by elcorbs — APPROVED** [review]

**Pull request merged**